packages/design-tokens/tokens/color.ts

```diff
@@ -1,16 +1,10 @@
 import * as colorTokens from "./color.json"
```

Comment:
@ActuallyACat what do you think about removing these neighbouring `*.ts` files within the tokens directory?

Reply:
Since our preferred non-SCSS consumption is to use `useTheme`

Reply:
I'd definitely like to see it go. Adding to this, it might also be useful to stop distributing the `json` token files for the same reason. I've had a peek at the monoliths and I don't think making this change would be terribly controversial.

Reply:
So, i agree 100% we don't need the json tokens.
I removed them and started to see the effects:
- The use of these json tokens within components in Kaizen wasn't too hard to refactor, but it makes this PR increase in size.
- The stylelint plugin uses these files to know about Kaizen tokens, and fixing this bloated the PR even further, as it had a few trickle down effects such as requiring extra Jest configuration to be able to parse `*.scss` files (remember how we we're parsing `color.scss` etc at first within the stylelint plugin, and it needed that special rule in Jest?). There are alternative methods but it's still going to bloat this.

How do you feel about making another breaking change after this one?

Reply:
Oh just to clarify...this doesn't relate to `tokens/*.ts` files. They're in the 🗑️ .

Reply:
Omg brainwave:

![image](https://user-images.githubusercontent.com/7651503/129136865-5f8a947e-6444-4a9b-ab86-17d995d57477.png)

![image](https://user-images.githubusercontent.com/7651503/129136947-03675618-8c98-4057-a02f-e3758f4e989f.png)


![image](https://user-images.githubusercontent.com/7651503/129136910-ddc14dc3-1eb5-4934-8b75-96f4ef645195.png)

![image](https://user-images.githubusercontent.com/7651503/129136922-5bc4af37-df35-45fe-a815-d5eed6e9a756.png)

Reply:
Love that! That's certainly preferable if we're removing the .json tokens in a future breaking change.